released/packages/coq-monae/coq-monae.0.3/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+
+homepage: "https://github.com/affeldt-aist/monae"
+dev-repo: "git+https://github.com/affeldt-aist/monae.git"
+bug-reports: "https://github.com/affeldt-aist/monae/issues"
+license: "LGPL-2.1-or-later"
+
+synopsis: "Monads and equational reasoning in Coq"
+description: """
+This Coq library contains a hierarchy of monads with their laws used
+in several examples of monadic equational reasoning."""
+
+build: [
+  [make "-j%{jobs}%"]
+  [make "sect5"]
+  [make "-C" "impredicative_set"]
+]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.13" & < "8.14~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.13~") }
+  "coq-mathcomp-fingroup" { (>= "1.12.0" & < "1.13~") }
+  "coq-mathcomp-algebra" { (>= "1.12.0" & < "1.13~") }
+  "coq-mathcomp-solvable" { (>= "1.12.0" & < "1.13~") }
+  "coq-mathcomp-field" { (>= "1.12.0" & < "1.13~") }
+  "coq-mathcomp-analysis" { (>= "0.3.4" & < "0.4~") }
+  "coq-infotheo" { >= "0.3" & < "0.4~"}
+  "coq-paramcoq" { >= "1.1.2" & < "1.2~" }
+]
+
+tags: [
+  "keyword:monae"
+  "keyword:effect"
```

Comment:
Please use `"keyword:effects"` to avoid duplicating keywords (see https://coq.inria.fr/opam/www/).

Also, it might be good to add `"date:yyyy-mm-dd"`, with the date of the release (not today, unless the release happened today).

Reply:
I have pushed force an update with those fixes. Thank for the review work.